### PR TITLE
bpo-9694: Fix misleading phrase "optional arguments"

### PR DESCRIPTION
**Pull request by rhettinger**
https://bugs.python.org/issue9694

I still need to change the examples in the main docs and add an introductory sentence or two explaining the terminology.  Will wait until further discussions are complete before making those edits.

### CODE REVIEW

File: Doc/howto/argparse.rst [hunk 1]
```diff
@@ -468,7 +468,7 @@ verbosity argument (check the output of ``python --help``)::
        print(answer)
 
 We have introduced another action, "count",
-to count the number of occurrences of a specific optional arguments:
+to count the number of occurrences of a specific options:
```
Comment (ericvsmith) [inline Doc/howto/argparse.rst]: The old version has this issue, too: shouldn't this be singular "options", or maybe "occurrences of specific options" (omitting "a")?

File: Misc/NEWS.d/next/Library/2020-12-19-12-33-38.bpo-9694.CkKK9V.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Argparse help no longer use the confusing phrase, "optional arguments".
```
Comment (ericvsmith) [inline Misc/NEWS.d/next/Library/2020-12-19-12-33-38.bpo-9694.CkKK9V.rst]: I think you should add: ', it now uses "options".'

**Review by ericvsmith — COMMENTED** [review]
Sorry I'm leaving this as two reviews: I pressed the wrong button on the first one.

Two little nits, otherwise looks good.

File: Doc/howto/argparse.rst (hunk 1 above)
Comment (rhettinger) [reply Doc/howto/argparse.rst]: Yes, that makes sense.

File: Misc/NEWS.d/next/Library/2020-12-19-12-33-38.bpo-9694.CkKK9V.rst (hunk 2 above)
Comment (rhettinger) [reply Misc/NEWS.d/next/Library/2020-12-19-12-33-38.bpo-9694.CkKK9V.rst]: Will do.

**Pull request merged**